src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/HtmlFormattingTest.cs

```diff
@@ -262,5 +264,38 @@ public class Foo
 }
 ");
         }
+
+        [Fact]
+        public async Task FormatsComponentTags()
```

Comment:
I'd add a test where there is > 1 component nesting with C#

Reply:
Will do. I just reached out to @pranavkm for a more realistic scenario.